autoload/ctrlsf/backend.vim

```diff
@@ -115,6 +115,9 @@ func! s:BuildCommand(args) abort
     " path
     call extend(tokens, ctrlsf#opt#GetPath())
 
+    " ignore stderr
+    call add(tokens, " 2>/dev/null")
```

Comment:
`>/dev/null` is not likely to work on Windows, I prefer `shellredir` to do the job.